src/components/NavHeader/NavHeader.styled.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+export const NavHeaderWrapperFull = styled.section`
+  position: relative;
+  width: 100%;
+  top: ${(props) => props.theme.spacing.zero};
+  background-color: ${(props) => props.theme.color.primary};
+  padding: ${(props) => props.theme.spacing.xSmall} ${(props) => props.theme.spacing.zero};
+`;
+
+export const NavHeaderWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: ${(props) => props.theme.spacing.zero} ${(props) => props.theme.spacing.medium};
+  margin: ${(props) => props.theme.spacing.zero} ${(props) => props.theme.spacing.base};
+
+  @media screen and (max-width: 425px) {
+    width: 70%;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  a:hover {
+    text-decoration: underline;
+    color: black;
+  }
```

Comment:
```suggestion
  a {
    text-decoration: none;
    
    &:hover {
      text-decoration: underline;
      color: black;
    }
  }
```

also it would be nice to use a variable for color :)